lib/src/message/trailer.rs

```diff
@@ -0,0 +1,211 @@
+// git-dit - the distributed issue tracker for git
+// Copyright (C) 2016, 2017 Matthias Beyer <mail@beyermatthias.de>
+// Copyright (C) 2016, 2017 Julian Ganz <neither@nut.email>
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+
+use message::line::{Line, Lines};
+use std::collections::LinkedList;
+
+#[derive(Debug, Hash, Eq, PartialEq, Ord, PartialOrd, Clone)]
+pub struct TrailerKey(String);
```

Comment:
Alright. Also, just to make sure the correct terminology is used.

Newtype => `struct Foo(String)`
Type Alias => `type Foo = String`